controllers/space/space_controller_test.go

```diff
@@ -42,228 +42,259 @@ func TestCreateSpace(t *testing.T) {
 	require.NoError(t, err)
 	base1nsTier := tiertest.Base1nsTier(t, tiertest.CurrentBase1nsTemplates)
 	t.Run("success", func(t *testing.T) {
-		// given
```

Comment:
It seems that there are a lot of changes in this test but it's actually almost the same test but wrapped with a table for two usecases:
- with a feature toggle annotation in the Space
- without any